Makefile

```
@@ -476,10 +476,12 @@ endif
 
 # OSX
 ifeq ($(NATIVE), osx)
-  ifneq ($(CLANG), 0)
-    OSX_MIN = 10.7
-  else
-    OSX_MIN = 10.5
+  ifneq ($(OSX_MIN),)
+    ifneq ($(CLANG), 0)
+      OSX_MIN = 10.7
```

```suggestion
      OSX_MIN = 10.12
```
Given the slowdowns (lack of optimization) required for anything prior.

---

I remember Kevin turned optimisation back on for old Mac OS X in a PR some days ago.